package.json

```diff
@@ -43,10 +43,10 @@
     "cors": "2.8.4",
     "express": "4.16.2",
     "extend": "3.0.1",
-    "fxa-shared": "1.0.4",
     "fxa-geodb": "0.0.8",
+    "fxa-shared": "1.0.4",
```

Comment:
@shikhar-scs just reset the git changes made to `package.json` and the shrinkwrap file above. Your PR is not changing any dependencies

Reply:
@vladikoff yeah sure will do